### ISSUE TITLE
The head cannot be fully recognized

### ISSUE THREAD

**Issue by gsc1456** [KwaiVGI/LivePortrait#122]
The generated video cannot fully identify the hair, headwear or hat, and will be distorted, with half moving and half still.

**Comment by cleardusk** [comment]
Yes, it’s a known issue! One of the possible reasons is that the training model does not include, or has very few, adornments such as hats. @gsc1456
